app/overrides/add_http_proxy_form.rb

```diff
@@ -0,0 +1,4 @@
+Deface::Override.new(:virtual_path => "http_proxies/_form",
+                     :name => "add_update_setting_input",
+                     :insert_bottom => '#primary',
+                     :partial => 'overrides/http_proxies/update_setting_input')
```

Comment:
While using deface is less intrusive, I think we should add a pagelet support in the `http_proxies/_form` file, like we did for the host form: https://github.com/theforeman/foreman/blob/48718f719b6282670d1ee1d0f36777ef99934f1a/app/views/hosts/_form.html.erb#L81

Reply:
Created https://github.com/theforeman/foreman/pull/10372